myfitnesspal/client.py

```diff
@@ -486,5 +509,21 @@ def _get_water(self, date):
 
         return Volume(ml=value)
 
+    def save_weight(self, weight):
+        result = self._post_request_for_url(
+            parse.urljoin(
+                self.BASE_URL,
+                '/measurements/save'
+            ),
+             {'authenticity_token':self.authenticity_token, 'weight[display_value]':weight}
```

Comment:
The indentation on this looks a little bit off; could you make sure that it's starting on the right column, and that it's fewer than 80 chars in length?